qa/rpc-tests/README.md

```diff
@@ -1,10 +1,8 @@
 Regression tests
 ================
 
-### [python-bitcoinrpc](https://github.com/jgarzik/python-bitcoinrpc)
-Git subtree of [https://github.com/jgarzik/python-bitcoinrpc](https://github.com/jgarzik/python-bitcoinrpc).
-Changes to python-bitcoinrpc should be made upstream, and then
-pulled here using git subtree.
+### [test_framework](test_framework)
+This contains some files of the [python-bitcoinrpc by jgarzik](https://github.com/jgarzik/python-bitcoinrpc).
```

Comment:
@jonasschnelli @jgarzik I think it's only authproxy.py

So I addressed that, I guess.